src/execution/buildFieldPlan.ts

```diff
@@ -0,0 +1,152 @@
+import { getBySet } from '../jsutils/getBySet.js';
+import { isSameSet } from '../jsutils/isSameSet.js';
+
+import type { FieldDetails } from './collectFields.js';
+
+export interface DeferUsage {
```

Comment:
This same type is also defined in `collectFields.ts`?

Reply:
cff34775 thanks!